pylatex/table.py

```diff
@@ -378,6 +378,77 @@ class Tabu(Tabular):
     packages = [Package('tabu')]
 
 
+    def __init__(self, table_spec, data=None, pos=None, *,
+                 row_height=None, col_space=None, width=None, booktabs=None,
+                 spread=None, to=None, **kwargs):
+        """
+        Args
+        ----
+        table_spec: str
+            A string that represents how many columns a table should have and
+            if it should contain vertical lines and where.
+        pos: list
+        row_height: float
+            Specifies the heights of the rows in relation to the default
+            row height
+        col_space: str
+            Specifies the spacing between table columns
+        booktabs: bool
+            Enable or disable booktabs style tables. These tables generally
+            look nicer than regular tables. If this is `None` it will use the
+            value of the ``booktabs`` attribte from the `~.active`
+            configuration. This attribute is `False` by default.
+        spread: str
+            Specifies the Tabu table should add a given amount of 'padding' to
+            the width of the table. This should be a latex dimension; for example:
+            "0 pt" or "1in"
+        to: str
+            Specifies the Tabu table should extend to a given width. This should be
+            a latex dimension; for example '4in'
+        width: int
+            The amount of columns that the table has. If this is `None` it is
+            calculated based on the ``table_spec``, but this is only works for
+            simple specs. In cases where this calculation is wrong override the
+            width using this argument.
+
+        References
+        ----------
+        * https://en.wikibooks.org/wiki/LaTeX/Tables#The_tabular_environment
+        """
+
+
+
+        super().__init__(table_spec, data, pos,
+                         row_height=row_height, col_space=col_space,
+                         width=width, booktabs=booktabs, **kwargs)
+
+        self._preamble = ""
+        if spread:
+            self._preamble = "spread " + spread
```

Comment:
Isn't it possible to just prepend these two new options to `self.data` here? That would make the dumps changes below with their hardcoded lengths not necessary anymore.